akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -538,6 +541,8 @@ private[persistence] trait Eventsourced extends Snapshotter with PersistenceStas
    * Common receive handler for processingCommands and persistingEvents
    */
   private abstract class ProcessingState extends State {
+    override def recoveryRunning: Boolean = false
+
```

Comment:
Why was this change needed? It's not referenced anywhere in the other changes as far as I can see.